packages/ios_platform_images/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.1.0+3
```

Comment:
Please change this to 0.1.1.  The "+N" nomenclature is for changes that don't affect the binary.

Reply:
I made the change for you so we can get this landed.

Reply:
@gaaclarke 👍 Seems other plugin causes CI red.